src/storage/btree.c

```diff
@@ -24922,6 +25300,7 @@ btree_range_scan_advance_over_filtered_keys (THREAD_ENTRY * thread_p, BTREE_SCAN
 		  ASSERT_ERROR_AND_SET (error_code);
 		  return error_code;
 		}
+	      COMMON_PREFIX_PAGE_SIZE_RESET (bts);
 	      /* Advance to next node. */
 	      pgbuf_unfix (thread_p, bts->C_page);
```

Comment:
The order below seems more appropriate for reviewing code.
```suggestion
	      /* Advance to next node. */
	      pgbuf_unfix (thread_p, bts->C_page);
	      COMMON_PREFIX_PAGE_SIZE_RESET (bts);
```

Reply:
The purpose is to express my opinion, so if you read it, please resolve it.